libs/langchain-community/src/document_loaders/fs/docx.ts

```diff
@@ -53,3 +119,15 @@ async function DocxLoaderImports() {
     );
   }
 }
+
+async function DocLoaderImports() {
+  try {
+    const WordExtractor = await import("word-extractor");
+    return WordExtractor.default;
+  } catch (e) {
+    console.error(e);
+    throw new Error(
+      "Failed to load word-extractor. Please install it with eg. `npm install word-extractor`."
```

Comment:
No way to just use `mammoth` is there?

Reply:
[No](https://github.com/mwilliamson/mammoth.js/issues/2#issuecomment-23871676), `mammoth` can only handle `.docx`